### ISSUE TITLE
Modal Width

### ISSUE THREAD

**Issue by WillianGruber** [Megabit/Blazorise#439]
Is there a way to increase the modal width further than Size="ModalSize.ExtraLarge" in ModalContent?

Thank you,

**Comment by stsrki** [comment]
I think your best option is to modify Bootstrap css class `.modal-xl` to increase the width.

**Comment by WillianGruber** [comment]
I would like not to change the bootstrap class. Maybe with attribute splatting next release it is possible. Will test it next realease

**Comment by stsrki** [comment]
Attribute splatting is already possible. It was released with 0.8.7

**Comment by stsrki** [comment]
@WillianGruber Have you tested attribute splatting?

**Comment by WillianGruber** [comment]
Not yet. I'm busy generating dynamic screens with RenderTreeFragment and Blazorise components.
Will look at this issue again in some days

**Comment by WillianGruber** [comment]
I managed to increase the width changing the max-width in this tag:
![image](https://user-images.githubusercontent.com/56033692/72348820-ff66d380-36b9-11ea-9e65-63c5334a9fe0.png)
Unfortunatelly it is innaccessible through splatting.

**Comment by stsrki** [comment]
I see. So basically the reason why splatting is not working is because I usually apply it to the tag that have `ClassNames`. And in this case it's the inner tag for bootstrap implementation.

```cshtml
@inherits BootstrapBase.BaseBootstrapModalContent
<div class="@DialogClassNames" role="document">
    <div class="@ClassNames" @attributes="@Attributes">
        @ChildContent
    </div>
</div>
```
I will need to see what can be done to apply it to the first tag also.

**Comment by stsrki** [comment]
One option is to steal some of the keys(stributes) from `Attributes` dictionary and apply it to the right place. In this case the steal the max-width.

Better option it so add `MaxWidth` property and set it to the right place.

**Comment by stsrki** [comment]
I plan to refactor Modal, so this ticket will be done in the next milestone.

#973

**Comment by stsrki** [comment]
Should be closed with the #3103. Instead of having a new size, I decided it is a better option to have fullscreen mode. Inspired by the https://getbootstrap.com/docs/5.1/components/modal/#fullscreen-modal
